etcd/server.sls

```diff
@@ -24,27 +20,33 @@ etcd_service:
 
 /etc/kubernetes/manifests/etcd.manifest:
   file.managed:
-    - source: salt://etcd/files/etcd.manifest
-    - template: jinja
-    - user: root
-    - group: root
-    - mode: 644
-    - makedirs: true
-    - dir_mode: 755
+  - source: salt://etcd/files/etcd.manifest
```

Comment:
I preffer 2-space indent for lists which is IMHO correct YAML approach, while both are working fine.